app/services/menu/loader.rb

```diff
@@ -43,8 +43,8 @@ def self.load
       Manager.map :top_menu do |menu|
         menu.sub_menu :monitor_menu,    :caption => N_('Monitor') do
           menu.item :dashboard,         :caption => N_('Dashboard')
-          menu.item :reports,           :caption => N_('Reports'),
-                    :url_hash => {:controller => '/reports', :action => 'index', :search => 'eventful = true'}
+          menu.item :reports,           :caption => N_('Config Mgmt. Reports'),
```

Comment:
Perhaps add a divider ("Reports") and then list "Config management" below it?  It's still not terribly pretty.